examples/rust/components/http-keyvalue-watcher/src/lib.rs

```diff
@@ -0,0 +1,135 @@
+use crate::bindings::wasi::http::outgoing_handler;
+use crate::bindings::wasi::http::types::*;
+use wasmcloud_component::wasi::logging::logging::{log, Level};
+mod bindings {
+    use super::KvWatcher;
+    wit_bindgen::generate!({ generate_all });
+    export!(KvWatcher);
+}
+
+use crate::bindings::exports::wasi::keyvalue::watcher::Guest as KvWatcherDemoGuest;
+
+#[derive(Debug)]
+struct KvWatcher {}
+
+impl KvWatcherDemoGuest for KvWatcher {
+    fn on_set(bucket: bindings::wasi::keyvalue::store::Bucket, key: String, value: Vec<u8>) {
+        let val = String::from_utf8(value.clone()).unwrap();
```

Comment:
Nit: This value doesn't need to be cloned and can just be consumed by the `from_utf8`